### ISSUE TITLE
Manifest doesn't allow for other plugin options

### ISSUE THREAD

**Issue by rjmreis** [hapijs/glue#51]
Not sure I'm doing everything right here, but while upgrading to v3 it seems I can't have other plugin options other than select. This is an example of what I'm trying to load:

```
{
  "connections": [
    {
      "port": 8080,
      "labels": ["http"]
    }
  ],
  "registrations": [
    {
      "plugin": "vision",
      "options": {
        "select": "http"
      }
    },
    {
      "plugin": "visionary",
      "options": {
        "select": "http",
        "engines": {
          "html": "handlebars"
        },
        "path": "/views"
      }
    }
  ]
}
```

**Comment by rjmreis** [comment]
It appears that the following syntax works. Hopefully this will help someone.

```
{
  "plugin": {
    "register": "visionary",
    "options": {
      "select": "http",
      "engines": {
        "html": "handlebars"
      },
      "path": "/views"
    }
  }
}
```

**Comment by gergoerdosi** [comment]
@rjmreis: I doubt it works, `select` should not be in `options`. In your case `select` was probably ignored and `visionary` was registered to all connections. I made an attempt to fix this in #49.

**Comment by rjmreis** [comment]
You're right @gergoerdosi, had to move it out of the plugin object. This worked:

```
{
  "plugin": {
    "register": "visionary",
    "options": {
      "engines": {
        "html": "handlebars"
      },
      "path": "views"
    }
  },
  "options": {
    "select": ["http"]
  }
}
```
